src/utils/lookup-table-utils.ts

```diff
@@ -170,16 +170,18 @@ export const validateLookupTableLanguages = async (
 
   try {
     logger.debug(`Checking descriptions and notes are different between languages`);
-    const duplicateDescriptionRows = await quack.all(`
-      SELECT "${joinColumn}", description, COUNT(language) as lang_count
-      FROM (SELECT * FROM "${lookupTableName}" where description IS NOT NULL)
-      GROUP BY description, "${joinColumn}" HAVING lang_count > 1
-    `);
-    const duplicateNoteRows = await quack.all(`
-      SELECT "${joinColumn}", notes, COUNT(language) as lang_count
-      FROM (SELECT * FROM "${lookupTableName}" WHERE notes IS NOT NULL)
-      GROUP BY notes, "${joinColumn}" HAVING lang_count > 1
-    `);
+    const duplicateDescriptionRows: TableData[] = [];
+    const duplicateNoteRows: TableData[] = [];
+    // const duplicateDescriptionRows = await quack.all(`
```

Comment:
Intensionally commented out while I consider alternative approaches to replace this code.